build/lib/rollup.ts

```diff
@@ -0,0 +1,109 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fs from 'fs';
+import * as rollup from 'rollup';
+import * as path from 'path';
+
+// getting around stupid import rules
```

Comment:
Let's not use "stupid" here

(also out of curiosity, what was the issue here?)

Reply:
I'm just restoring code that was incorrectly deleted during previous merge.  though I agree that we can remove that word.